src/converter/style.py

```diff
@@ -380,7 +380,7 @@ def convert_effects(fig_node: dict) -> _Effects:
                 )
             )
 
-        elif e["type"] == "FOREGROUND_BLUR":
+        elif e["type"] == "FOREGROUND_BLUR" or e["type"] == "BACKGROUND_BLUR":
```

Comment:
The only difference between blur conversion for these two was the `type` so I've unified the conversion to keep things DRY